resources/metadata/fortify.yaml

```
@@ -382,8 +382,7 @@ spec:
       default: "/download/currentStateFprDownload.html"
     - name: defaultVersioningModel
       type: string
-      description: "The default project versioning model used in case `projectVersion` parameter is empty for
-        creating the version based on the build descriptor version to report results in SSC, can be one of `'major'`,
+      description: "The default project versioning model used for creating the version based on the build descriptor version to report results in SSC, can be one of `'major'`,
```

The content is fine, but I need to scroll horizontally now to see all of the diff.
```suggestion
      description: "The default project versioning model used for creating the version based on the build descriptor
      version to report results in SSC, can be one of `'major'`, `'major-minor'`, `'semantic'`, `'full'`"
```

---

> We should take care that the description displays nicely when calling `piper`.
> That is the face to the users ...

Fully agreed. What I mean is just the formatting in the YAML. It will be processed and not reflect in the CLI, if it is declared as above in my suggestion. Line-breaks will be replaced by spaces.